lib/tether/toolbox.go

```diff
@@ -35,8 +36,12 @@ import (
 type Toolbox struct {
 	*toolbox.Service
 
-	config *ExecutorConfig
-	stop   chan struct{}
+	sess struct {
+		sync.Mutex
```

Comment:
There are two locks we should be taking:
1. Lock for the `config.Sessions` map (this doesn't currently exist)
2. When we operate on a session we should be locking `session.m`

Adding this lock doesn't address (1) which still needs to happen. Once we have (1) there's no need for this caching.